pkg/network/tracer/connection/tracer.go

```diff
@@ -557,7 +557,6 @@ func populateConnStats(stats *network.ConnectionStats, t *netebpf.ConnTuple, s *
 		Application: protocols.Application(s.Protocol_stack.Application),
 		Encryption:  protocols.Encryption(s.Protocol_stack.Encryption),
 	}
-	stats.StaticTags |= uint64(s.Conn_tags)
```

Comment:
why did you remove that?

Reply:
https://github.com/DataDog/datadog-agent/pull/16940/files#r1194171920